cpp/src/parquet/schema-test.cc

```diff
@@ -1911,13 +1911,13 @@ TEST_F(TestTemporalSchemaElementConstruction, TemporalCases) {
 
   std::vector<SchemaElementConstructionArguments> timestamp_cases = {
       {"timestamp_T_ms", LogicalType::Timestamp(true, LogicalType::TimeUnit::MILLIS),
-       Type::INT64, -1, true, ConvertedType::TIMESTAMP_MILLIS, true, check_TIMESTAMP},
-      {"timestamp_F_ms", LogicalType::Timestamp(false, LogicalType::TimeUnit::MILLIS),
        Type::INT64, -1, false, ConvertedType::NA, true, check_TIMESTAMP},
+      {"timestamp_F_ms", LogicalType::Timestamp(false, LogicalType::TimeUnit::MILLIS),
+       Type::INT64, -1, true, ConvertedType::TIMESTAMP_MILLIS, true, check_TIMESTAMP},
```

Comment:
These tests verify that `ConvertedType::TIMESTAMP_*` is set regardless of timezone awareness